raiden_contracts/tests/unit/test_recover_from_signature.py

```diff
@@ -23,6 +23,8 @@ def test_verify(
         create_channel,
         create_balance_proof,
 ):
+    """ the signature test contract returns the correct addresses on the balance hash signed
+    by both ends of a channel """
```

Comment:
Please follow the docstring rules from https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md.
E.g.:
```
    """ Check if ECVerify.ecverify returns the correct address

    This checks whether ECVerify.ecverify returns the correct addresses on the balance hash signed
    by both ends of a channel.
    """
```

Reply:
Done this here and there!